test/unit/sync/branch.test.ts

```diff
@@ -5,10 +5,11 @@ import { mocked } from "ts-jest/utils";
 import { Subscription } from "../../../src/models";
 import { Application } from "probot";
 import { createWebhookApp } from "../../utils/probot";
-import createJob from "../../setup/create-job";
 import { processInstallation } from "../../../src/sync/installation";
 import nock from "nock";
 import {getLogger} from "../../../src/config/logger";
+import {Hub} from "@sentry/types/dist/hub";
+import {BackfillMessagePayload} from "../../../src/sqs/backfill";
 
 jest.mock("../../../src/models");
 
```

Comment:
Commit, branch and pullrequest tests are skipped here btw,
We need to think if we want to keep them or delete them.